skyxplore-ui-test/src/test/java/com/mentoring/pageobject/CharacterSelectionPage.java

```diff
@@ -141,6 +141,20 @@ public CharacterSelectionPage clickRenameCharacter(String characterName) {
         return this;
     }
 
+    @Step("Rename from: {characterNameOld} - and choose new character: {characterNameNew}.")
+    public OverviewPage renameAndChooseCharacter(String characterNameOld, String characterNameNew) {
+        clickRenameCharacter(characterNameOld);
+        renameCharacter(characterNameNew);
+        return selectCharacter(characterNameNew);
+    }
+
+    @Step("")
```

Comment:
Please fill the step description or just remove: clickForDeleteCharacterButtonInList is already annotated with Step. If you decide to have them both annotated, then please re-think the descriptions (one is about just clicking, the other is actually performing the deletion)